harold/plugins/deploy.py

```diff
@@ -748,7 +745,7 @@ def hold(self, irc, sender, channel, *reason):
             return
 
         type = DeployHoldType.manual
-        if 'freeze' in reason.lower():
+        if 'freeze' in reason[0].lower():
```

Comment:
`*reason` makes `reason` a tuple (forgot about this).

```
Type "help", "copyright", "credits" or "license" for more information.
>>> def test_function(*args):
...     print(type(args))
... 
>>> test_function(5)
<type 'tuple'>
>>> test_function('abd')
<type 'tuple'>
>>> 
```

Reply:
I think you want to look for "freeze" among all the words in the reason not just the first, right? This would only catch `harold hold freeze` but not `harold hold code freeze`

Reply:
Fixed.